### ISSUE TITLE
Add multiple: bad regex

### ISSUE THREAD

**Issue by blauertee** [hsdevelops/cron-telebot#24]
The following regex does not match the following crontab:
https://github.com/hsdevelops/cron-telebot/blob/c483fcee976a1427ca608b22ce2aac419f0493ba/common/utils.py#L27-L34

```
0 10 * 4-7 2 Clean up a table
0 10 * 4-7 4 Check the calendar
0 14 * 4-7 5 Check this and that and that
```

I was trying to run something from April to July in this case. Generally I think trying to validate the cron statement here via regex is unnecessary since it's checked by 
https://github.com/hsdevelops/cron-telebot/blob/c483fcee976a1427ca608b22ce2aac419f0493ba/bot/actions/actions.py#L170-L172
anyway and seems to be [pretty complex](https://stackoverflow.com/questions/14203122/create-a-regular-expression-for-cron-statement).

I would maybe just use a very simple regex like `^(\S+\s+\S+\s+\S+\s+\S+)\s+(.+)$` And leave the further checking to the library. Maybe you could even just use the 
https://github.com/hsdevelops/cron-telebot/blob/c483fcee976a1427ca608b22ce2aac419f0493ba/bot/actions/actions.py#L284
function in `add_jobs` to be coherent when adding jobs. 

If you think what I say makes sense I could implement it that way :)

Thanks for this very useful and lightweight bot!

**Comment by huishun98** [comment]
Thanks for raising this issue! Feel free to create a PR, will review and merge accordingly.

**Comment by blauertee** [comment]
I hope I did fix it [here](https://github.com/blauertee/cron-telebot). I don't feel comfortable raising a pull request yet since I wasn't able to run pytest. I ran into some dependency issues and then my time was too limited to really look int it. 

But feel free to do that for me :)

**Comment by huishun98** [comment]
Thanks, I've referenced the commits and fixed this issue in b8e40fbcc2a18d9e100ab0d7ff34e7e532e1dd52. Discarded regex as we're no longer doing validity checks here.
